_blog/2023-12-06-using-axum-rust.mdx

```diff
@@ -45,17 +46,17 @@ struct Message {
 }
 
 enum ApiResponse {
-    Ok,
+    OK,
```

Comment:
Nit: enum variants [should be UpperCamelCase](https://github.com/shuttle-hq/www/pull/224/files).

Reply:
Hey O thanks for the code review!

I am a little confused - is this not already UpperCamelCase? Unless it should be `JSONData` instead of JsonData (for example)

Reply:
It should be Ok not OK, JsonData is correct. It's a minor detail though, sorry for the nitpick! 😅

Reply:
Oh whoops sorry :grimacing: I just checked on the Rust playground and it looks like it does compile. I guess if we need to come back to this article at any point I can update it then